ui-v2/app/styles/components/definition-table/layout.scss

```diff
@@ -0,0 +1,16 @@
+%definition-table > dl {
+  position: relative;
+}
+%definition-table > dl > dt {
+  position: absolute;
+}
+%definition-table > dl > dt {
+  width: 140px;
+}
+%definition-table > dl > dd {
+  padding-left: 140px;
+}
+%definition-table > dl > * {
+  min-height: 1em;
+  margin-bottom: 0.4em;
+}
```

Comment:
Now we are using plain `dl`s for other things. I moved what was our 'default layout' for `dl`s to a 'definition table' component. This means we shouldn't have to keep overwriting `dl` margins/padding anymore.